collectd-cpu/docs/cpu.interrupt.md

```diff
@@ -0,0 +1,12 @@
+
+---
+title: cpu.interrupt
+brief: CPU time spent while servicing hardware interrupts -- A hardware interrupt happens at the physical layer. When this occurs, the CPU will stop whatever else it is doing and service the interrupt. This metric measures how many jiffies were spent handling these interrupts. In order to get a percentage this value must be compared against the sum of all CPU states. A sustained high value for this metric may be caused by faulty hardware such as a broken peripheral.
```

Comment:
Are you making the Brief descriptions extra long for a reason? Is it because that's what will show up in the new metric finder? (i.e. per discussion with rt and/or joe d?) Because it will make tables like these take up an enormous amount of room in the docs. https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.couchdb.html#metrics

Reply:
Went ahead and shortened the brief field to only the first sentence.

Reply:
So I'm not sure that was the right thing to do (which is why I posed the original question). MFinder descriptions can be 1024 characters, and more info in the UI is probably a good thing, even if it makes the docs ugly. Did you move the additional info to the description section, so it will show up in the docs? (If not in the MFinder). I guess no need to change back, but I'd like to have @beccatortell weigh in at some point.

Reply:
I made it so that the `brief` field will be the first sentence of the main description that goes at the bottom of these metric markdown files.  So all of the original content will be present in the markdown under the main text at the bottom.  I don't really understand what uses brief vs. what uses long description with the new metric finder stuff.